.github/workflows/ci.yml

```diff
@@ -59,6 +60,12 @@ jobs:
             -DCMAKE_INSTALL_PREFIX="$WORKSPACE_INSTALL_PATH" \
             -DCMAKE_PREFIX_PATH="$SOFA_ROOT/lib/cmake" \
             "
+          if [[ "${{ matrix.qp_solver }}" == "proxqp" ]]; then
+          cmake_options="$cmake_options \
+            -DSOFTROBOTSINVERSE_ENABLE_PROXQP=ON \
+            -DSOFA_ALLOW_FETCH_DEPENDENCIES=ON \
```

Comment:
```suggestion
            -DSOFA_ALLOW_FETCH_DEPENDENCIES=ON \
```
Still needed?

Reply:
It does not yet, still working on the PR as it is a draft